src/migrate/stub/ts.stub

```diff
@@ -1,15 +1,15 @@
 import * as Knex from "knex";
 
-exports.up = function (knex: Knex): Promise<any> {
+export async function up(knex: Knex): Promise<Knex.SchemaBuilder> {
```

Comment:
I don't thing tat typing is right. AFAIK Migration does not always resolve with Knex.SchemaBuilder instance.

Reply:
true, isn't always return `Knex.SchemaBuilder`, but the default template use `createTable` and `dropTable` which both returns `Knex.SchemaBuilder`

Reply:
updated